### PR TITLE
Marriage abroad in Turkey: two outcomes

### PR DESCRIPTION
**Pull request by pmanrubia**
### Trello card
* [Add two outcomes to Turkey in Marriage abroad](https://trello.com/c/9f78ZxfR/569-3-marriage-abroad-improvement-turkey-2-outcomes) 

## Description
Reduces to two outcomes Turkey

## Factcheck 
[Factcheck URL](https://smart-answers-pr-2958.herokuapp.com/marriage-abroad/y/turkey)

## Expected changes

* [URL on GOV.UK](https://www.gov.uk/marriage-abroad/y/turkey)

### Before

image

### After

image

### CODE REVIEW

**Comment by pmanrubia** [comment]
Thank you @leenagupte I have addressed your comments

**Comment by leenagupte** [comment]
@pmanrubia Did you intend to squash everything into one commit?

**Comment by pmanrubia** [comment]
Sorry @leenagupte, not sure what happened here. I have reworked the PR again.

File: test/integration/smart_answer_flows/marriage_abroad_test.rb [hunk 1]
```diff
@@ -886,9 +884,7 @@ def self.translations
   context "ceremony in Turkey, resident in Turkey, partner other" do
```
Comment (leenagupte) [inline test/integration/smart_answer_flows/marriage_abroad_test.rb]: It's a minor thing, but the title of this test should probably be changed to reflect that it's testing the same sex route, and the test above does with opposite sex.

File: lib/smart_answer_flows/marriage-abroad.rb [hunk 2]
```diff
@@ -361,7 +361,6 @@ def define
       outcome :outcome_opposite_sex_marriage_in_south_korea
       outcome :outcome_opposite_sex_marriage_in_slovakia
       outcome :outcome_opposite_sex_marriage_in_thailand
-      outcome :outcome_opposite_sex_marriage_in_turkey
```
Comment (leenagupte) [inline lib/smart_answer_flows/marriage-abroad.rb]: This line should probably be removed in the following commit (7506402f8cfd9dfef3e6ce7b26f9a8c60be7eac1) where the extraneous `turkey` code is removed.

**Comment by leenagupte** [comment]
@pmanrubia Two very minor follow-up comments.

File: lib/smart_answer_flows/marriage-abroad.rb (hunk 2 above)
Comment (pmanrubia) [inline lib/smart_answer_flows/marriage-abroad.rb]: It seems that If this is removed in the following commit, the tests would not be green.

File: test/integration/smart_answer_flows/marriage_abroad_test.rb (hunk 1 above)
Comment (pmanrubia) [inline test/integration/smart_answer_flows/marriage_abroad_test.rb]: Good spot @leenagupte 
I have addressed your comment.

File: lib/smart_answer_flows/marriage-abroad.rb (hunk 2 above)
Comment (leenagupte) [inline lib/smart_answer_flows/marriage-abroad.rb]: Ah! That makes sense.

**Pull request merged**